### PR TITLE
JBPM-9281 Multiple error notification popups while adding new deploym…

### PR DESCRIPTION
**Pull request by bxf12315**
…ent unit

**Thank you for submitting this pull request**

**JIRA**: [JBPM-9281](https://issues.redhat.com/browse/JBPM-9281)

**Referenced Pull Requests**:
* paste the link(s) from GitHub here
* link 2
* link 3 etc.

**Business Central**: [WAR file](https://donwnload.here/something)

**VS Code**: [plugin](https://donwnload.here/something)

<pre>
To retest a PR or trigger a specific build please add a comment:

* Retest PR: <b>jenkins retest this</b>
* A full downstream build: <b>jenkins do fdb</b>
* A compile downstream build: <b>jenkins do cdb</b>
* A full production downstream build: <b>jenkins do product fdb</b>
* An upstream build: <b>jenkins do upstream</b>
</pre>

### CODE REVIEW

File: kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-client/src/main/java/org/kie/workbench/common/screens/server/management/client/wizard/NewContainerWizard.java [hunk 1]
```diff
@@ -199,4 +203,9 @@ public void pageSelected(final int pageNumber) {
         }
         super.pageSelected(pageNumber);
     }
+
+    //just for test duplicate WizardPageSelectedEvent
```
Comment (nmirasch) [inline kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-client/src/main/java/org/kie/workbench/common/screens/server/management/client/wizard/NewContainerWizard.java:207]: @bxf12315 remove the comment

**Review by nmirasch — APPROVED** [review]
@bxf12315  just a minor thing related with a commented line, but changes look good to me.

File: kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-client/src/main/java/org/kie/workbench/common/screens/server/management/client/wizard/NewContainerWizard.java (hunk 1 above)
Comment (bxf12315) [reply kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-client/src/main/java/org/kie/workbench/common/screens/server/management/client/wizard/NewContainerWizard.java]: Done

**Comment by prashuts** [comment]
Jenkins execute full downstream build

**Comment by bxf12315** [comment]
jenkins do fdb

**Review by prashuts — APPROVED** [review]

**Pull request merged**